src/main/java/de/adorsys/keycloak/config/model/RealmImport.java

```diff
@@ -27,12 +27,15 @@
 import org.springframework.stereotype.Component;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 
 @Component
 public class RealmImport extends RealmRepresentation {
     private List<AuthenticationFlowImport> authenticationFlowImports;
 
+    private List<HashMap<String, Object>> userProfile;
```

Comment:
```suggestion
    private List<LinkedHashMap<String, Object>> userProfile;
```
The order of entry should be kept as is.